client/tokenized-payment-request/transformers/wc-to-stripe.js

```diff
@@ -13,26 +13,37 @@ import { __ } from '@wordpress/i18n';
 export const transformCartDataForDisplayItems = ( cartData ) => {
 	const displayItems = cartData.items.map( ( item ) => ( {
 		amount: parseInt( item.prices.price, 10 ),
-		// TODO: should we also add variation attributes?
-		label: [ item.name, item.quantity > 1 && ` (x${ item.quantity })` ]
+		label: [
+			item.name,
+			item.quantity > 1 && `(x${ item.quantity })`,
+			item.variation &&
+				item.variation
+					.map(
+						( variation ) =>
+							`${ variation.attribute }: ${ variation.value }`
+					)
+					.join( ', ' ),
+		]
 			.filter( Boolean )
-			.join( '' ),
-		pending: true,
+			.join( ' ' ),
 	} ) );
 
-	if ( cartData.totals.total_tax ) {
+	const taxAmount = parseInt( cartData.totals.total_tax || '0', 10 );
+	if ( taxAmount ) {
 		displayItems.push( {
-			amount: parseInt( cartData.totals.total_tax, 10 ),
+			amount: taxAmount,
 			label: __( 'Tax', 'woocommerce-payments' ),
-			pending: true,
```

Comment:
Removed `pending` to allow for the itemized amounts to be displayed - I checked the previous implementation, and they also weren't using it

Reply:
Out of curiosity: do we happen to know what the business value of the `pending` property is? Which state does it represent?

Reply:
It seems to be used by browsers to display things in different ways.
[Stripe says:](https://docs.stripe.com/js/appendix/payment_item_object)
> If you might change this amount later (for example, after you have calcluated shipping costs), set this to true

In our case, it seems that `pending: true` will never be needed because there's always going to be an address selected on the Google Pay/Apple Pay modal. And if an address is selected, we'll also have the shipping options available from the Store API.